### PR TITLE
Major rewording for Ghost version 1 

### PR DESCRIPTION
**Pull request by pascalandy**
It's all in plain english :)

### CODE REVIEW

File: ghost/content.md [hunk 1]
```diff
@@ -1,33 +1,58 @@
...
 ```
 
-This will start a Ghost instance listening on the default Ghost port of 2368.
+### Custom port
 
 If you'd like to be able to access the instance from the host without the container's IP, standard port mappings can be used:
 
 ```console
-$ docker run --name some-ghost -p 8080:2368 -d ghost
+$ docker run —d name ghostdemo -p 3001:2368 ghost
```
Comment (SeerUK) [inline ghost/content.md]: Should this still be `--name`?

**Comment by pascalandy** [comment]
Sure! Will fix this!

File: ghost/content.md [hunk 2]
```diff
@@ -1,33 +1,58 @@
...
+```console
+$ docker run —d --name ghostdemo -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost/ ghost:0.11-alpine
+```
+
+### Breaking change 
+
+If you want to run Ghost 0.11.xx, be aware of the container’s path difference:
+
+- Ghost 1.x.x is: `/var/lib/ghost/content/`
+- Ghost 0.11.x is: ``/var/lib/ghost/`
```
Comment (yosifkit) [inline ghost/content.md]: Extra backtick here.

File: ghost/content.md [hunk 3]
```diff
@@ -1,33 +1,58 @@
...
 
 %%LOGO%%
 
 # How to use this image
 
+This will start a Ghost instance listening on the default Ghost port of 2368.
+
 ```console
-$ docker run --name some-ghost -d ghost
+$ docker run —d --name ghostdemo ghost
```
Comment (SISheogorath) [inline ghost/content.md]: Without published ports, this isn't going to listen anywhere. you should add `-p 2368:2368`

File: ghost/content.md [hunk 4]
```diff
@@ -1,33 +1,58 @@
...
+### Breaking change 
+
+If you want to run Ghost 0.11.xx, be aware of the container’s path difference:
+
+- Ghost 1.x.x is: `/var/lib/ghost/content/`
+- Ghost 0.11.x is: ``/var/lib/ghost/` 
+
+# Sqlite Database
+
+This Docker image for Ghost use Sqlite. There is nothing special to configure.
```
Comment (yosifkit) [inline ghost/content.md]: Should be "uses SQLite" (and "SQLite" in the header).

File: ghost/content.md [hunk 5]
```diff
@@ -1,33 +1,58 @@
...
+
+# Docker Volume
+
 Alternatively you can use a [data container](http://docs.docker.com/engine/tutorials/dockervolumes/) that has a volume that points to `/var/lib/ghost` and then reference it:
```
Comment (yosifkit) [inline ghost/content.md]: This should also be adjusted: 
> a volume that points to `/var/lib/ghost/content` (or `/var/lib/ghost` for `0.11`)

**Comment by yosifkit** [comment]
@pascalandy, Thanks for the PR!

File: ghost/content.md [hunk 6]
```diff
@@ -1,33 +1,58 @@
 # Ghost
 
-Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT License, designed to simplify the process of online publishing for individual bloggers as well as online publications.
+Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT Licence, designed to simplify the process of online publishing for individual bloggers as well as online publications.
```
Comment (tianon) [inline ghost/content.md]: Uhh, "License" is the correct spelling here (see the wikipedia link). :smile:

File: ghost/content.md (hunk 5 above)
Comment (pascalandy) [inline ghost/content.md]: LGTM

File: ghost/content.md (hunk 4 above)
Comment (pascalandy) [inline ghost/content.md]: LGTM

File: ghost/content.md (hunk 2 above)
Comment (pascalandy) [inline ghost/content.md]: LGTM

File: ghost/content.md (hunk 3 above)
Comment (tianon) [inline ghost/content.md]: Any reason to bother changing this example when the new version is functionally identical? (also `—d` and `-d` are not the same :smirk:)

File: ghost/content.md [hunk 7]
```diff
@@ -1,33 +1,58 @@
...
 ```
 
-This will start a Ghost instance listening on the default Ghost port of 2368.
+### Custom port
 
 If you'd like to be able to access the instance from the host without the container's IP, standard port mappings can be used:
 
 ```console
-$ docker run --name some-ghost -p 8080:2368 -d ghost
+$ docker run —d --name ghostdemo -p 3001:2368 ghost
```
Comment (tianon) [inline ghost/content.md]: Any particular reason to change this example too? (and the port?)

File: ghost/content.md (hunk 3 above)
Comment (pascalandy) [inline ghost/content.md]: It states that it would run with the default port 2368. It think we are fine.

File: ghost/content.md [hunk 8]
```diff
@@ -1,33 +1,58 @@
...
+# Stateful
 
-You can also point the image to your existing content on your host:
+Mount your existing content. In this example we also use the Alpine base image.
+
+### Ghost 1.x.x
 
 ```console
-$ docker run --name some-ghost -v /path/to/ghost/blog:/var/lib/ghost ghost
+$ docker run —d --name ghostdemo -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost/content/ ghost:1-alpine
```
Comment (tianon) [inline ghost/content.md]: Please drop the trailing slash on `content/` and stick to the `some-ghost` naming (we use that convention fairly consistently across the official images descriptions).

File: ghost/content.md [hunk 9]
```diff
@@ -1,33 +1,58 @@
...
 ```
 
+### Ghost 0.11.xx
+
+```console
+$ docker run —d --name ghostdemo -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost/ ghost:0.11-alpine
```
Comment (tianon) [inline ghost/content.md]: Again, `—d` and `-d` are not the same (this won't copy-paste into a terminal properly).

**Comment by pascalandy** [comment]
@yosifkit thanks for all your work over the Docker hub :)

File: ghost/content.md [hunk 10]
```diff
@@ -1,33 +1,58 @@
...
+```
+
+### Breaking change 
+
+If you want to run Ghost 0.11.xx, be aware of the container’s path difference:
```
Comment (tianon) [inline ghost/content.md]: Please use `'` (standard apostrophe) instead of `’` here.

File: ghost/content.md (hunk 3 above)
Comment (pascalandy) [inline ghost/content.md]: > This will start a Ghost instance listening on the default Ghost port of 2368.

Comment (SeerUK) [inline ghost/content.md]: While that is true to an extent @SISheogorath is correct, it wouldn't be published - if you went directly to the IP of the container on the Docker network interface it would be available on port 2368, however this is not how you usually access containers as that IP address will be completely unreliable, and will only be accessible locally by default.

File: ghost/content.md (hunk 4 above)
Comment (tianon) [inline ghost/content.md]: It _can_ be configured to use an external database, though, so i think this is a little misleading (makes it sound like _only_ `SQLite` is supported).  With 1.x, it can even be configured entirely via environment variables (https://docs.ghost.org/docs/config#section-running-ghost-with-config-env-variables).

File: ghost/content.md (hunk 6 above)
Comment (pascalandy) [inline ghost/content.md]: Crap my corrector is messing with me.

Comment (pascalandy) [inline ghost/content.md]: > (also —d and -d are not the same 😏)

My markdown editor is messing with these characters ... Will fix!

File: ghost/content.md (hunk 7 above)
Comment (pascalandy) [inline ghost/content.md]: 1. The port, too many apps uses 8080 so I decided to use 3001. Purely random and not use in any containers I use.

2. I don't like `some-ghost` because I can't double click on it. I also prefer the see the word ghost first

3. `-d` I prefer to see it prior the name. Unwritten rule for me.

File: ghost/content.md (hunk 8 above)
Comment (pascalandy) [inline ghost/content.md]: 1. OK, I'll drop the `/`
2. I'll revert back to some-ghost

File: ghost/content.md (hunk 3 above)
Comment (yosifkit) [inline ghost/content.md]: There is the example of publishing a port on the next example.  I almost never publish a port locally since I'd rather not publish my services to my local network, but I run a local DNS service that lets me hit containers by name.

**Comment by pascalandy** [comment]
Gents, I think we got something solid for Ghost 1 here.

File: ghost/content.md [hunk 11]
```diff
@@ -8,30 +8,55 @@ Ghost is a free and open source blogging platform written in JavaScript and dist
...
 ```
 
-Alternatively you can use a [data container](http://docs.docker.com/engine/tutorials/dockervolumes/) that has a volume that points to `/var/lib/ghost` and then reference it:
+### Ghost 0.11.xx
+
+```console
+$ docker run -d --name some-ghost -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost ghost:0.11-alpine
+```
+
+### Breaking change
```
Comment (yosifkit) [inline ghost/content.md]: Remove trailing whitespace.

File: ghost/content.md [hunk 12]
```diff
@@ -8,30 +8,55 @@ Ghost is a free and open source blogging platform written in JavaScript and dist
...
+```console
+$ docker run -d --name some-ghost -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost ghost:0.11-alpine
+```
+
+### Breaking change 
+
+If you want to run Ghost 0.11.xx, be aware of the container’s path difference:
+
+- Ghost 1.x.x is: `/var/lib/ghost/content`
+- Ghost 0.11.x is: `/var/lib/ghost`
```
Comment (yosifkit) [inline ghost/content.md]:
```diff
-- Ghost 1.x.x is: `/var/lib/ghost/content`
-- Ghost 0.11.x is: `/var/lib/ghost` 
+-	Ghost 1.x.x is: `/var/lib/ghost/content`
+-	Ghost 0.11.x is: `/var/lib/ghost`
```

File: ghost/content.md (hunk 11 above)
Comment (pascalandy) [inline ghost/content.md]: LGTM

File: ghost/content.md (hunk 12 above)
Comment (pascalandy) [inline ghost/content.md]: LGTM

File: ghost/content.md (hunk 10 above)
Comment (pascalandy) [inline ghost/content.md]: I learn my lesson. Can't write technical stuff in IA writer :-p

**Comment by pascalandy** [comment]
Something else I can do ?

**Comment by yosifkit** [comment]
https://github.com/docker-library/docs/commit/6a32373189131daead1d0d6a2b94306570d13c6d
Merged manually (and somehow disconnected it from the PR) :astonished:
